tests/helper_functions.py

```diff
@@ -41,6 +42,15 @@ def add_bundle(store: Store, bundle: dict) -> None:
         store.session.add(bundle_obj)
         store.session.commit()
 
+    @staticmethod
+    def add_archive(store: Store, file_id: int, archive_task_id:int=1234) -> None:
```

Comment:
```suggestion
    def add_archive(store: Store, file_id: int, archive_task_id:int) -> None:
```
I think you made a fixture for this further up